app/controllers/miq_policy_controller.rb

```diff
@@ -574,33 +586,40 @@ def replace_right_cell(nodetype, replace_trees = [], presenter = nil)
         if @profiles
           r[:partial => 'profile_list']
         elsif @policies || (@view && @sb[:tree_typ] == 'policies')
-          right_cell_text = _("All %{typ} %{model}") % {:typ => "#{ui_lookup(:model => @sb[:nodeid])} #{@sb[:mode] ? @sb[:mode].capitalize : ""}", :model => ui_lookup(:models => "MiqPolicy")}
+          right_cell_text = _("All %{typ} %{model}") % {
+            :typ   => "#{ui_model_from_id(@sb[:nodeid])} #{@sb[:mode] ? @sb[:mode].capitalize : ""}",
```

Comment:
You mean add `.try(:camelize)` everywhere and get rid of `ui_model_from_id` function?
Possible.

I kinda like "ui_model_from_id" expressing more intent than "camelize".
I feel it's hard to keep track of where we use what casing, and pretty impossible to read (or write) this code and notice "Oh, this .titleize should have been .camelize", which lead these bugs in the first place.